File: fizzbuzz/fizzbuzz.c
#include <stdio.h>

int
main (void) {
    for (int n = 1; n <= 100; n++) {
        if (n % 15 == 0)
            printf("fizzbuzz\n");
        else if (n % 5 == 0)
            printf("buzz\n");
        else if (n % 3 == 0)
            printf("fizz\n");
        else
            printf("%d\n", n);
    }
    return 0;
}

/* EOF */

File: classics/fizzbuzz/fizzbuzz.R
#!/usr/bin/Rscript

cat(paste(
    c(sapply(1:100, function(n) {
        ifelse(n %% 15 == 0, "fizzbuzz",
        ifelse(n %%  5 == 0, "buzz",
        ifelse(n %%  3 == 0, "fizz", n)))
    }), ""),
    collapse="\n")
)

# EOF
